hw4/04-templates.js

```diff
@@ -12,6 +12,54 @@ app.set('view engine', 'pug');
 const url = 'https://restcountries.com/v3.1/all';
 
 // Add your code here
+const data = [];
+const getData = (url) => {
+  axios
+    .get(url)
+    .then((response) => {
+      response.data.forEach((country) => {
+        data.push(country);
+      });
+    })
+    .catch(() => {
+      console.log('An error occured. Please reload the page.');
+    });
+};
+
+getData(url);
+
+const getList = (route) => {
```

Comment:
I like this solution to check each route and do something different depending on the route